MdeModulePkg/Universal/Variable/RuntimeDxe/Variable.c

```diff
@@ -53,6 +53,13 @@ VARIABLE_INFO_ENTRY  *gVariableInfo = NULL;
 ///
 BOOLEAN  mEndOfDxe = FALSE;
 
+// MU_CHANGE [BEGIN] - 279849
+///
+/// The flag to indicate if a Reclaim has been done at Runtime.
+///
+BOOLEAN  mReclaimedAtRuntime = FALSE;

```

Comment:
FYI, this is to mitigate the number of runtime reclaim for wear leveling sake. In case you want to update the commit message.

Reply:
I'll go ahead and update the commit message to include that information / history